### PR TITLE
Review feedback

### PR DESCRIPTION
**Pull request by lightclient**
Piper left some good comments on the PR into the eth1 spec: https://github.com/ethereum/eth1.0-specs/pull/219

### CODE REVIEW

**Review by voith — APPROVED** [review]
LGTM

File: src/eth1spec/evm/__init__.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ class Evm:
 
     pc: Uint
     stack: List[U256]
-    memory: bytes
+    memory: bytearray
```
Comment (SamWilsn) [inline src/eth1spec/evm/__init__.py:48]: Would `Bytes` be even better?

**Review by SamWilsn — APPROVED** [review]
Looks good enough!

File: src/eth1spec/evm/__init__.py (hunk 1 above)
Comment (lightclient) [reply src/eth1spec/evm/__init__.py:48]: `Bytes` is `bytes` though :(

Comment (SamWilsn) [reply src/eth1spec/evm/__init__.py:48]: For type checking, `bytes` is a superclass of `bytearray` IIUC

Comment (lightclient) [reply src/eth1spec/evm/__init__.py:48]: o

**Pull request merged**